dpk_connector/Makefile

```diff
@@ -0,0 +1,52 @@
+# Use make help, to see the available rules
+REPOROOT=..
+include $(REPOROOT)/.make.defaults
+
+TAG := "v${DPK_LIB_VERSION}"
```

Comment:
What is this (i.e. TAG) used for?  I don't see any use anywhere.

Reply:
Removed.